src/test/server.c

```diff
@@ -125,6 +137,17 @@ void *pdispatch(void *data)
         printf("In server callback for msg #%ld\n", msg->hinfo.cookie);
     }
 
+    if (!authorize(msg->hinfo.target, msg->hinfo.instance))
```

Comment:
The instance field is to authorize the IO request. We had talked about a compromised qemu process trying to send an IO on a vdisk that it does not own. The authorize function will use information populated during the provisioning of the vdisk to check if the instance is allowed to do IO on this vdisk. The true implementation of authorize in the vxhs server will take care o this. This will also ensure that remote IOs for resiliency are authorized in a similar manner.